Podcast/InternalProfileViewController.swift

```diff
@@ -80,6 +80,9 @@ class InternalProfileViewController: UIViewController, UITableViewDelegate, UITa
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         // Move to view here
         switch sectionsAndItems[indexPath.section][indexPath.row] {
```

Comment:
This looks absurd, can you assign row and height to variables and then switch on those two instead of indexing sections and items every time?